R/read_habitatdata.R

```diff
@@ -42,16 +42,17 @@
 #'   The variable \code{certain} will be \code{FALSE} if
 #'   the original code consists of 2 or 3 possible vegetation types, and \code{TRUE}
 #'   if only one vegetation type is provided.
-#'     \item Some polygons contain both a standing water habitat type
-#'     and \code{rbbmr}:
-#'     \code{3130_rbbmr},
-#'     \code{3140_rbbmr},
-#'     \code{3150_rbbmr} and
-#'     \code{3160_rbbmr}.
-#'     Since \code{habitatmap_stdized_2020_v1}, the two vegetation
-#'     types \code{31xx} and \code{rbbmr} are split up and one row is
-#'     created for each of them.
-#'     The variable \code{certain} in this case will be \code{TRUE} for both types.
+#'   \item Some polygons contain both a standing water habitat type
+#'   and \code{rbbmr}:
+#'   \code{3130_rbbmr},
+#'   \code{3140_rbbmr},
+#'   \code{3150_rbbmr} and
+#'   \code{3160_rbbmr}.
+#'   Since \code{habitatmap_stdized_2020_v1}, the two vegetation
+#'   types \code{31xx} and \code{rbbmr} are split up and one row is
+#'   created for each of them.
+#'   The variable \code{certain} in this case will be \code{TRUE} for both
+#'   types.
```

Comment:
Following [this](https://github.com/inbo/n2khab/pull/140#discussion_r630850765) and [this](https://github.com/inbo/n2khab/pull/140#discussion_r630966051):

```suggestion
#'   Since \code{habitatmap_stdized_2020_v1}, the two vegetation
#'   types \code{31xx} and \code{rbbmr} are split up and one row is
#'   created for each of them, with \code{phab} for each new row simply
#'   set to the original value of \code{phab}. 
#'   The variable \code{certain} in this case will be \code{TRUE} for both
#'   types.
#'   \item After those first two steps, a given polygon could contain the 
#'   same type with the same value for \code{certain} repeated several
#'   times, e.g. when \code{31xx_rbbmr} is present with \code{phab} = yy\%
#'   and \code{31xx} is present with \code{phab} = zz\%.
#'   In that case the rows with the same \code{polygon_id}, \code{type} 
#'   and \code{certain} were gathered into one row and the respective 
#'   \code{phab} were added up.
```

Reply:
Thanks, will commit it. I'll drop 'vegetation' from 'vegetation types' and then update the Rd file in another commit.

Reply:
Note: I see that there is quite inconsistent usage of 'vegetation' throughout read_habitatdata.R (this includes watersurfaces). Will try to rectify asap (mostly by dropping or replacing vegetation), in a separate commit.

Reply:
Edit: in a separate _pull request_